build.gradle

```diff
@@ -0,0 +1,198 @@
+import static org.apache.tools.ant.taskdefs.condition.Os.*
+
+buildscript {
+    dependencies {
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.1'
+    }
+}
+
+plugins {
+    id 'com.github.ksoichiro.console.reporter' version '0.5.0'
+}
+
+apply plugin: 'java'
+apply plugin: 'jacoco'
+apply plugin: 'signing'
+apply plugin: 'maven'
+apply plugin: 'idea'
```

Comment:
why do we need idea ?

Reply:
because otherwise intellij doesn't pick up the generated protobuf files when you're working inside this project. this of course doesn't hold if you import the lib as a gradle dependency. i can remove this if we don't want that functionality

Reply:
i don't think we need it since we won't change it often.